crawler_mcp/core/connection_pool.py

```
@@ -0,0 +1,320 @@
+
+            # Remove old connection
+            if old_conn in self.connections:
+                self.connections.remove(old_conn)
+            if old_conn in self.connection_health:
+                del self.connection_health[old_conn]
+
+            # Try to close old connection
+            try:
+                await old_conn.close()
+            except Exception:
+                pass  # Ignore close errors
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)